modules/test/playwright/pages/dynamic-data-mapping-form-web/FormBuilderSidePanelPage.ts

```diff
@@ -71,7 +75,9 @@ export class FormBuilderSidePanelPage {
 
 		await this.paragraphFieldTextarea.press('End');
 
-		await this.page.waitForLoadState('networkidle');
+		await this.page.waitForResponse(
```

Comment:
Never reference localhost directly. Point to `apiHelpers.baseUrl` instead. Also, please, create an util function like the one described in https://github.com/liferay-bpm/liferay-portal/pull/4380#discussion_r2038118790.